### PR TITLE
Added maintainconnections key for ACF mail server

### PR DESCRIPTION
**Pull request by dandyfopper**
Allows ACF "Maintain connection to mail server" option to be configured

### CODE REVIEW

File: models/BaseAdobe.cfc [hunk 1]
```diff
@@ -697,6 +697,7 @@ component accessors=true extends='cfconfig-services.models.BaseConfig' {
 		if( !isNull( thisConfig.keyAlias ) ) { setMailSignKeyAlias( thisConfig.keyAlias ); }
 		if( !isNull( thisConfig.keypassword ) ) { setMailSignKeyPassword( passwordManager.decryptMailServer( thisConfig.keypassword ) ); }
 		if( !isNull( thisConfig.mailsentloggingenable ) ) { setMailLogEnabled( thisConfig.mailsentloggingenable ); }
+		if( !isNull( thisConfig.maintainConnections ) ) { setMailLogEnabled( thisConfig.maintainConnections ); }
```
Comment (bdw429s) [inline models/BaseAdobe.cfc:700]: You're setting this into the MailLogEnabled setting, which is incorrect. A new setting needs created in `CaseConfig.cfc` to store this.  I would call it `mailMaintainConnections`.

File: models/BaseAdobe.cfc [hunk 2]
```diff
@@ -1670,6 +1671,7 @@ component accessors=true extends='cfconfig-services.models.BaseConfig' {
 		if( !isNull( getMailSignKeyAlias() ) ) { thisConfig.keyAlias = getMailSignKeyAlias(); }
 		if( !isNull( getMailSignKeyPassword() ) ) { thisConfig.keypassword = passwordManager.encryptMailServer( getMailSignKeyPassword() ); }
 		if( !isNull( getMailLogEnabled() ) ) { thisConfig.mailsentloggingenable = getMailLogEnabled() ? true : false; }
+		if( !isNull( getMaintainConnections() ) ) { thisConfig.maintainConnections = getMaintainConnections() ? true : false; }
```
Comment (bdw429s) [inline models/BaseAdobe.cfc:1674]: You're calling `getMaintainConnections()`, but never created that setting.  See my suggested name in the other comment.  Notice all the mail settings generally start with the word "mail".  Also, if there is a chance the key won't already exist in the user's config XML file, use bracket notation to ensure we create it with the correct case.
```
thisConfig[ 'maintainconnections' ] = ..
```

**Review by bdw429s — CHANGES_REQUESTED** [review]

File: models/BaseAdobe.cfc (hunk 1 above)
Comment (dandyfopper) [reply models/BaseAdobe.cfc]: followed your suggestion here - I wasn't paying enough attention on the initial pull request

**Comment by dandyfopper** [comment]
@bdw429s I think I have resolved the issues - having trouble figuring out how to test it locally though

**Pull request merged**